src/test/java/org/ambraproject/wombat/service/PeerReviewServiceTest.java

```diff
@@ -0,0 +1,124 @@
+            ))
+        ),
+        "10.1371/journal.pone.0207232.r002", ImmutableMap.of(
+            "doi", "10.1371/journal.pone.0207232.r001",
+            "itemType", "reviewLetter",
+            "files", ImmutableMap.of("letter", ImmutableMap.of(
+                "crepoKey", "info:doi/10.1371/journal.pone.0207232.r002.xml"
+            ))
+        ),
+        "10.1371/journal.pone.0207232.r003", ImmutableMap.of(
+            "doi", "10.1371/journal.pone.0207232.r003",
+            "itemType", "reviewLetter",
+            "files", ImmutableMap.of("letter", ImmutableMap.of(
+                "crepoKey", "info:doi/10.1371/journal.pone.0207232.r003.xml"
+            ))
+        )
+        );
+
+
+    PeerReviewService serviceWithMockedContent = new PeerReviewServiceImpl() {
+      @Override
+      String getReviewXml(Map<String, ?> metadata) throws IOException {
+        Map<String, ?> files = (Map<String, ?>) metadata.get("files");
+        Map<String, ?> letter = (Map<String, ?>) files.get("letter");
+        String crepoKey = (String) letter.get("crepoKey");
+
+        String letterContent = null;
+        if (crepoKey == "info:doi/10.1371/journal.pone.0207232.r001.xml") {
+          letterContent = "<?xml version=\"1.0\" encoding=\"utf-8\"?><sub-article specific-use=\"decision-letter\">" +
+              "<front-stub><custom-meta-group><custom-meta><meta-name>Submission Version</meta-name><meta-value>0</meta-value></custom-meta></custom-meta-group></front-stub>" +
+              "<body>InitialDecisionLetterSampleBody</body></sub-article>";
+        }
+        if (crepoKey == "info:doi/10.1371/journal.pone.0207232.r002.xml") {
+          letterContent = "<?xml version=\"1.0\" encoding=\"utf-8\"?><sub-article article-type=\"author-comment\">" +
+              "<front-stub><custom-meta-group><custom-meta><meta-name>Submission Version</meta-name><meta-value>1</meta-value></custom-meta></custom-meta-group></front-stub>" +
+              "<body>FirstRoundAuthorResponseSampleBody</body></sub-article>";
+        }
+        if (crepoKey == "info:doi/10.1371/journal.pone.0207232.r003.xml") {
+          letterContent = "<?xml version=\"1.0\" encoding=\"utf-8\"?><sub-article specific-use=\"decision-letter\">" +
+              "<front-stub><custom-meta-group><custom-meta><meta-name>Submission Version</meta-name><meta-value>1</meta-value></custom-meta></custom-meta-group></front-stub>" +
+              "<body>FirstRoundDecisionLetterSampleBody</body></sub-article>";
+        }
+        return letterContent;
+      }
+    };
+
+    String html = serviceWithMockedContent.asHtml(itemTable);
+    assertThat(html, containsString("Original Submission"));
+    assertThat(html, containsString("InitialDecisionLetterSampleBody"));
+    assertThat(html, containsString("Revision 1"));
+//    assertThat(html, containsString("FirstRoundAuthorResponseSampleBody"));
+    assertThat(html, containsString("FirstRoundDecisionLetterSampleBody"));
+
```

Comment:
getAllReviewsXml() in PeerReviewServiceImpl is pretty long. it would be great to have tests which exercise all branches... I'm not sure that current test coverage does that.